peekingduck/pipeline/nodes/draw/mosaic_bbox.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Mosaics area bounded by bounding boxes over detected object
+"""
+
+from typing import Any, Dict, List
+import cv2
+import numpy as np
+
+from peekingduck.pipeline.nodes.node import AbstractNode
+
+
+class Node(AbstractNode):  # pylint: disable=too-few-public-methods
+    """Mosaics areas bounded by bounding boxes on image.
+
+    The mosaic bbox node helps to anonymize detected objects by pixelating the
+    areas bounded by bounding boxes in an image.
+
+    Inputs:
+
+        |img|
+
+        |bboxes|
+
+    Outputs:
+        |img|
+
+    Configs:
+        blocks (:obj:`int`): **default = 7**
+            controls the intensity of pixelation. Reducing the number of blocks
+            increases the level of opacity but decreases the number of pixels.
```

Comment:
Would it be clearer if we rename this config as `pixelation` or `pixelation_degree`? Perhaps can look at conventions for naming. I think the description can be improved too.

Reply:
Regarding the naming of the mosaic config, would this be better?

Config Name:
mosaic_level: 7

Description:
defines the resolution of a mosaic filter (width x height). The number corresponds to the number of rows and columns used to create a mosaic. For example, the default setting (mosaic_level: 7) creates a 7 x 7 mosaic filter. Increasing the number increases the intensity of pixelation over an area.